server/server.go

```diff
@@ -194,6 +200,11 @@ func (s *Server) SetSealer(sealer Sealer) {
 	s.sealer = sealer
 }
 
+// RegisterRPC publishes in the server the methods on rcvr.
+func (s *Server) RegisterRPC(rcvr interface{}) error {
+	return s.dispatcher.Register(rcvr)
+}
```

Comment:
I'm not sure I understand the question. RegisterRPC is called by the server, with APIs that it would like to expose to clients throug the keyless protocol.